dipy/tracking/local/localtracking.py

```diff
@@ -93,12 +100,16 @@ def _generate_streamlines(self):
             directions = dg.initial_direction(s)
             directions = directions[:max_cross]
             for first_step in directions:
-                stepsF = local_tracker(dg, tc, s, first_step, vs, F, ss, fixed)
-                if stepsF < 0:
+                stepsF, tissue_class = local_tracker(dg, tc, s, first_step, vs, F, ss, fixed)
+                if not (self.return_all or
+                        tissue_class == 2 or  # ENDPOINT
```

Comment:
Is there a way to access ```cdef enum TissueClass````from here? (defined in dipy.tracking.local.tissue_classifier.pyd)